bootstrap.go

```diff
@@ -0,0 +1,123 @@
+package p2pd
+
+import (
+	"context"
+	"errors"
+	"math/rand"
+	"time"
+
+	inet "github.com/libp2p/go-libp2p-net"
+	pstore "github.com/libp2p/go-libp2p-peerstore"
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+var BootstrapPeers = []string{
```

Comment:
Yeah, we should be able to support different bootstrap peers. 
I think what we want is a configuration file of sorts for the daemon profile, and there we can list bootstrap peers there.
But I left that for a subsequent pr (we can open issue for this).

Reply:
Personally I'd like to see an application "opening a session" with a daemon, and providing any initial seed configuration as part of that handshake. 

I think of the daemon as a service provided by the environment/OS for applications to use, versus a _sidecar_ for applications. As such, multi-tenancy and app-specific configuration should be part of the protocol.

For now, it's good that it's on our radar. An issue would be great, and perhaps a note in the specs.

Reply:
Yeah, that's not a bad idea at all.
Upon further thought I'll add a `-bootstrapPeers` flag so that we can pass a comma separated list of peers in the command line, so that we can punt on the configuration.

Reply:
Added a command-line flag in https://github.com/libp2p/go-libp2p-daemon/pull/11/commits/bd604c45561a403ee660e5ee07a6c72864204135